backend/vaa-strapi/src/api/app-customization/controllers/app-customization.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * app-customization controller
+ */
+
+import {factories} from '@strapi/strapi';
+import {API} from '../../../functions/utils/api';
+import {staticSettings} from 'vaa-shared';
+
+export default factories.createCoreController(API.AppCustomization, () => ({
```

Comment:
Translations and candidate app FAQ items are now preprocessed by the backend. This could also be done in the frontend but as the response can be cached, we can save resources by doing it here.